src/components/dropdown.tsx

```
@@ -74,7 +74,7 @@ function Dropdown({ size = "md", items }: DropdownProps) {
         aria-expanded={isOpen}
         aria-label="메뉴 열기"
         onClick={toggleDropdown}
-        className="flex h-24 w-24 items-center justify-center rounded-full bg-white"
+        className="bg-white flex h-24 w-24 items-center justify-center rounded-full"
```

ESLint

[error] Replace `bg-white·flex·h-24·w-24·items-center·justify-center·rounded-full` with `flex·h-24·w-24·items-center·justify-center·rounded-full·bg-white`

(prettier/prettier)